utilities/src/ordered_set.rs

```diff
@@ -60,74 +58,100 @@ impl<T: Ord> OrderedSet<T> {
 
 	/// Clear the set
 	pub fn clear(&mut self) {
-		self.0.clear();
+		self.0 = BoundedVec::default();
 	}
 }
 
-impl<T: Ord> From<Vec<T>> for OrderedSet<T> {
-	fn from(v: Vec<T>) -> Self {
+impl<T: Ord, S: Get<u32>> From<BoundedVec<T, S>> for OrderedSet<T, S> {
+	fn from(v: BoundedVec<T, S>) -> Self {
 		Self::from(v)
 	}
 }
 
+#[cfg(feature = "std")]
+impl<T, S> fmt::Debug for OrderedSet<T, S>
+where
+	T: fmt::Debug,
+	S: Get<u32> + fmt::Debug,
```

Comment:
I don't think we need this line?

Reply:
I can't seem to derive `Debug` and the code won't compile without it since it's needed for `assert` in the tests

Reply:
I mean is `S: Get<u32> + fmt::Debug,` required?

Reply:
You're right that wasn't needed